src/include/compat.h

```diff
@@ -33,6 +33,13 @@
 #define O_DSYNC O_SYNC
 #endif
 
+#ifndef EUCLEAN
+#define EUCLEAN         117	/* Structure needs cleaning */
```

Comment:
can we keep this in `src/key_value_store/kv_flat_btree_async.cc` ? as `EUCLEAN` is not used anywhere else .